src/CognitiveServices/CognitiveServices/Models/PSNetworkRuleSet.cs

```diff
@@ -0,0 +1,213 @@
+﻿using Microsoft.WindowsAzure.Commands.Common.Attributes;
```

Comment:
@yangyuan please add a license header

Reply:
@yangyuan this comment was not addressed

Reply:
@yangyuan ignore this -- there was a weird caching problem with GitHub that caused the old changes to be shown. This has been resolved.